nodeup/pkg/model/kube_controller_manager.go

```diff
@@ -157,18 +156,11 @@ func (b *KubeControllerManagerBuilder) buildPod() (*v1.Pod, error) {
 
 	// Ensure the Volume Plugin dir is mounted on the same path as the host machine so DaemonSet deployment is possible
 	if volumePluginDir == "" {
-		switch b.Distribution {
-		case distros.DistributionContainerOS:
-			// Default is different on ContainerOS, see https://github.com/kubernetes/kubernetes/pull/58171
-			volumePluginDir = "/home/kubernetes/flexvolume/"
-
-		case distros.DistributionCoreOS:
-			// The /usr directory is read-only for CoreOS
+		if b.IsKubernetesGTE("1.15") {
 			volumePluginDir = "/var/lib/kubelet/volumeplugins/"
-
-		default:
-			volumePluginDir = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
 		}
+
+		volumePluginDir = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
```

Comment:
Doesn't this need to be in an else block? Otherwise the previous if block has no effect.

Reply:
oh right true